eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/SchemaConfig.scala

```diff
@@ -12,6 +12,7 @@ final case class SchemaConfig(
   metaJournalTable: String = "metajournal",
   pointerTable: String = "pointer",
   pointer2Table: String = "pointer2",
+  snapshotTable: String = "snapshot",
```

Comment:
just a note, akka currently uses `snapshots` table. We need to be 100% to not conflict with it :)

Reply:
I did not think about it, to be honest. Other name is required here, indeed.

Reply:
How about "snapshot_buffer"?